apps/client/src/components/Layouts/BaseLayout.tsx

```diff
@@ -38,12 +39,16 @@ function BaseLayout({ children, checker, heading }: BaseLayoutProps) {
             }}
           >
             <Content>
-              {title && <FormTitle>{title}</FormTitle>}
+              {title && (
+                <FormTitle style={{ maxWidth: "inherit" }}>{title}</FormTitle>
```

Comment:
Inline style? Maybe create an atom instead?
Even better would be to fix FormTitle itself.

Reply:
Good point. I've changed this in ASC-UI in this [PR](https://github.com/Amsterdam/amsterdam-styled-components/pull/1086).